metricbeat/Makefile

```diff
@@ -51,7 +51,7 @@ configs: python-env
 .PHONY: imports
 imports:
 	mkdir -p include
-	python ${ES_BEATS}/metricbeat/scripts/generate_imports.py > include/list.go
+	python ${ES_BEATS}/metricbeat/scripts/generate_imports.py ${BEAT_DIR} > include/list.go
```

Comment:
The reason this doesn't work, is that BEAT_DIR has different meaning for community beats. We need an if / else to check if it is a community beat or not ...